components/script/dom/element.rs

```diff
@@ -1276,12 +1276,8 @@ impl Element {
     pub fn parse_fragment(&self, markup: DOMString) -> Fallible<Root<DocumentFragment>> {
         // Steps 1-2.
         let context_document = document_from_node(self);
-        let new_children = if context_document.is_html_document() {
-            ServoParser::parse_html_fragment(self, markup)
-        } else {
-            // FIXME: XML case
-            unimplemented!()
-        };
+        // TODO: XML case.
```

Comment:
Check if there is an issue; if not, file one. In either case, make this `FIXME(#NNNN): XML case.`